FlightReservation/components/modalComponent.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    View,
+    Text,
+    Image,
+    Modal,
+  } from 'react-native';
+import { AnimatedCircularProgress } from 'react-native-circular-progress';
+
+function ModalCustom(props) {
```

Comment:
Same as above, let's try to set a convention for naming the component and the it's files.